apiary-metastore-listener/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.expedia.apiary</groupId>
+    <artifactId>apiary-extensions-parent</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>apiary-metastore-listener</artifactId>
+  <name>Apiary Metastore Listener</name>
+  <description>Hive Metastore Listener for Apiary that forwards events on to AWS SNS</description>
+
+  <dependencies>
+    <!-- TODO: should we mark these as provided? or shade/relocate them? -->
```

Comment:
@rpoluri For the AWS jars that we require, how are we going to get them onto the classpath for the listener? I was thinking we could either:

1. Create an uber-jar for the listener and put them in there
2. Have something else put them on the classpath and then mark them as "provided" here. 

I think I have a slight preference for 1 as that ensures we have a more tightly controlled package that we deliver. Not sure what others feel about this - @patduin ? @teabot ? @mroark1m ?

Reply:
listeners run in hive metastore process, so we don't have to anything specific for listeners.

I am already handling that during metastore startup. https://github.com/ExpediaInc/apiary-metastore-docker/blob/master/files/startup.sh#L57

Reply:
Yeah, there are pros and cons to both approaches. I'm a bit concerned about adding the entire aws-java-sdk to the Hive classpath, that could potentially have some "fun" side effects". So my suggestion for now would be that we aim to have the metastore-listener use as few 3rd party libs as possible and we bundle them into a far jar.

Reply:
Yes, once we have this publishing to Maven Central we're going to need a PR on the other project to switch over to use this instead of what it's currently doing.

Reply:
look like you already sorted it but I am in favor of shading as well